examples/stats/exporter/stackdriver.js

```diff
@@ -0,0 +1,157 @@
+const {
+  StackdriverStatsExporter
+} = require("@opencensus/exporter-stackdriver");
+
+const fs = require("fs");
+const readline = require("readline");
+
+// Create the Stats manager
+const stats = new Stats();
+
+// [START setup_exporter]
+// Enable OpenCensus exporters to export metrics to Stackdriver Monitoring.
+// Exporters use Application Default Credentials to authenticate.
+// See https://developers.google.com/identity/protocols/application-default-credentials
+// for more details.
+// Add your project id to the Stackdriver options
+const exporter = new StackdriverStatsExporter({ projectId: "your-project-id" });
+
+// Pass the created exporter to Stats
+stats.registerExporter(exporter);
+// [END setup_exporter]
+
+// The latency in milliseconds
+const mLatencyMs = stats.createMeasureDouble(
+  "repl/latency",
+  MeasureUnit.MS,
+  "The latency in milliseconds per REPL loop"
+);
+
+// Counts/groups the lengths of lines read in.
+const mLineLengths = stats.createMeasureInt64(
+  "repl/line_lengths",
+  MeasureUnit.BYTE,
+  "The distribution of line lengths"
+);
+
+// Create a stream to read our file
+const stream = fs.createReadStream("./test.txt");
+
+// Create an interface to read and process our file line by line
+const lineReader = readline.createInterface({ input: stream });
+
+const tagKey = "method";
+
+// Register the view.
+const latencyView = stats.createView(
+  "demo/latency",
+  mLatencyMs,
+  AggregationType.DISTRIBUTION,
+  [tagKey],
+  "The distribution of the repl latencies",
+  // Latency in buckets:
+  // [>=0ms, >=25ms, >=50ms, >=75ms, >=100ms, >=200ms, >=400ms, >=600ms, >=800ms, >=1s, >=2s, >=4s, >=6s]
+  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
+);
+
+// Register the view.
+const lineCountView = stats.createView(
+  "demo/lines_in",
+  mLineLengths,
+  AggregationType.COUNT,
+  [tagKey],
+  "The number of lines from standard input"
+);
+
+// Register the view.
+const lineLengthView = stats.createView(
+  "demo/line_lengths",
+  mLineLengths,
+  AggregationType.DISTRIBUTION,
+  [tagKey],
+  "Groups the lengths of keys in buckets",
+  // Bucket Boudaries:
+  // [>=0B, >=5B, >=10B, >=15B, >=20B, >=40B, >=60B, >=80, >=100B, >=200B, >=400, >=600, >=800, >=1000]
+  [0, 5, 10, 15, 20, 40, 60, 80, 100, 200, 400, 600, 800, 1000]
+);
+
+// The begining of our REPL loop
+let startTime = new Date();
+let endTime;
+
+// REPL is the read, evaluate, print and loop
+lineReader.on("line", function(line) {
+  // Read
+  try {
+    const processedLine = processLine(line); // Evaluate
+    console.log(processedLine); // Print
+
+    // Registers the end of our REPL
+    endTime = new Date();
+
+    const tags = { method: "repl", status: "OK" };
+
+    stats.record({
+      measure: mLineLengths,
+      tags,
+      value: processedLine.length
+    });
+
+    stats.record({
+      measure: mLatencyMs,
+      tags,
+      value: endTime.getTime() - startTime.getTime()
```

Comment:
`startTime` === `endTime`. Could you please correct me if Im wrong?

Reply:
Currently yes, because we are not doing any fancy work in ```processLine()``` method.

Reply:
Used ```process.hrtime``` to record the latency value. PTAL